grid/chat/delete-message/maps/delete-message.ts

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable jest/no-export */
+
+import { RecordingProcessOptions, SuperfaceTest } from '@superfaceai/testing';
+
+export const deleteMessageTest = (
+  provider: string,
+  destination: string[],
```

Comment:
nitpick: you pass one valid and one invalid destination so I would type it as object with `valid` and `invalid` property to better communicate use of `destination`.